README.md

```diff
@@ -60,8 +60,7 @@ See lf documentation for more.
 - `lf_terminal_cmd` name of Kakoune command that will spawn terminal/tmux window
   with lf. It *must* expose `$kak_session` and `$kak_client` environmental variables.
   It also must set variable `KAKLF` to `yes`. By default it uses `hatch-terminal` command
-  from [Kakoune-extra]. I couldn't get it to work with build-in `terminal`
-  command, PRs welcome.
```

Comment:
this still applies. I'd prefer to get it working with builtin `terminal` command. Please don't remove this.

Reply:
Right, sorry, 3am jitters.
I actually played around with it, and it's really not hard to get it to work with `terminal`:
```
terminal sh -c "env KAKLF=yes kak_session=%val{client} kak_client=%val{cleint} lf"
```
I'll open a PR later